server/svix-server/src/main.rs

```diff
@@ -90,11 +102,12 @@ async fn main() {
         println!("Migrations run");
         exit(0);
     } else if let Some(Commands::Jwt {
-        command: JwtCommands::Generate,
+        command: JwtCommands::Generate { org_id },
     }) = &args.command
     {
+        let org_id = org_id.clone().unwrap_or_else(default_org_id);
```

Comment:
I was getting issues when trying to avoid the clone, and tbh, it doesn't really matter. It's a manual place that just runs once.

Reply:
```
else if let Some(Commands::Jwt {
        command: JwtCommands::Generate { org_id },
    }) = &mut args.command
  {
        let org_id = org_id.take().unwrap_or_else(default_org_id);
```

This might work to avoid the clone, but it looks a little funky

Reply:
Actually is there any reason you can't just take the whole command? It's the last in the chain.